core/storage/collection/gae_models.py

```diff
@@ -328,7 +328,7 @@ def compute_models_to_commit(
         )
 
         # Here we use cast because the additional_models is list of BaseModels
-        # and we want to hint the MyPy that this is CollectionRightsModel.
+        # and we want to hint to MyPy that this is CollectionRightsModel.
```

Comment:
Why can't we handle this in the typedefs at the top of the method?

Reply:
Fixed